src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx

```diff
@@ -995,23 +995,25 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
     return (
       <div className="data smart-wearable">
         {this.renderFields()}
-        <div className="required-permissions">
-          <Header sub>{t('create_single_item_modal.smart_wearable_permissions_label')}</Header>
-          <TagField value={requiredPermissions?.map(permission => permission.replaceAll('_', ' '))} search={false} disabled multiple />
-          <p className="learn-more">
-            {t('create_single_item_modal.required_permissions_learn_more_about', {
-              learn_more: (
-                <a
-                  href="https://docs.decentraland.org/creator/development-guide/sdk7/scene-metadata/#required-permissions"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  {t('global.learn_more')}
-                </a>
-              )
-            })}
-          </p>
-        </div>
+        {requiredPermissions?.length && (
```

Comment:
Would you mind using a ternary operator here too? Related to the previous comment in the `RightPanel.tsx` file.